scripts/arm_wrapper.sh

```diff
@@ -2,16 +2,18 @@
 
 DEVNAME=$1
 
+sleep 5 # allow the system enough time to load disc information such as title
+
+echo "[ARM] Starting ARM on ${DEVNAME}" | logger -t ARM -s
+/bin/su -l -c "echo /usr/bin/python3 /opt/arm/arm/ripper/main.py -d ${DEVNAME} | at now" -s /bin/bash arm
+
 # Check to see if the admin page is running, if not, start it
 if pgrep -f "runui.py" > /dev/null
 then
-        echo "ARM Webgui running  on " | logger -t ARM
+        echo "[ARM] ARM Webgui running. " | logger -t ARM -s
```

Comment:
the `-s` makes it also output to the console (for when you are running it manually)